neo/test/rawiotest/test_neuralynxrawio.py

```diff
@@ -134,6 +135,39 @@ def test_read_ncs_files_sideeffects(self):
         self.assertListEqual(rawio._sigs_t_start, [8408.806811, 8427.832053, 8487.768561])
         self.assertEqual(len(rawio._sigs_memmaps), 3)  # check only that there are 3 memmaps
 
+    def test_single_file_mode(self):
+        """
+        Tests reading of single files.
+        """
+
+        # test single analog signal channel
+        fname = self.get_filename_path('Cheetah_v5.6.3/original_data/CSC1.ncs')
+        rawio = NeuralynxRawIO(filename=fname)
+        rawio.parse_header()
+
+        self.assertEqual(rawio._nb_segment, 2)
+        self.assertEqual(len(rawio.ncs_filenames), 1)
```

Comment:
maybe check that `nse_filenames` and `nev_filenames` are empty since only the ncs file should be loaded?